pyproject.toml

```diff
@@ -12,7 +12,7 @@ license = 'GPL-3.0-only'
 authors = ['Andrew Sazonov', 'Simon Ward']
 readme = 'README.md'
 homepage = 'https://github.com/easyScience/easyDiffractionApp'
-repository = 'https://easyscience.apptimity.com/'
+repository = 'https://download.easydiffraction.org/'
```

Comment:
how do we pay for easyDiffraction.org? Is this domain in your name?
We should probably start keeping track of domains we/the group administers.